content/docs/command-reference/artifacts/get.md

```diff
@@ -0,0 +1,92 @@
+## artifacts get
+
+Download an artifact tracked in a DVC project into the current working
+directory.
+
+## Synopsis
+
+```usage
+usage: dvc artifacts get [-h] [-q | -v]
+                         [--rev [<version>]] [--stage [<stage>]]
+                         [-o [<path>]] [-j <number>] [-f]
+                         [--config CONFIG]
+                         [--remote REMOTE] [--remote-config [REMOTE_CONFIG ...]]
+                         url name
+
+positional arguments:
+  url                   Location of DVC repository to download from
+  name                  Name of artifact in the repository
+```
+
+## Description
+
+Provides an easy way to download artifacts tracked in a DVC project.
+`dvc artifacts get` supports downloading artifacts both from the
+[Studio Model Registry](/doc/studio/user-guide/model-registry) and from DVC
+remotes. Unlike `dvc get`, `dvc artifacts get` supports downloading an artifact
+by name, rather than by path.
```

Comment:
Minor, but I would flip the last two sentences. Also, let's mention that it supports downloading an artifact not only by name, but also by version and/or stage.